lib/ransack/helpers/form_helper.rb

```diff
@@ -197,8 +197,12 @@ def default_sort_order(attr_name)
             end
           end
 
+          def indicator?
+            @options.delete :hide_indicator unless Ransack.options[:hide_sort_order_indicators]
+          end
+
           def order_indicator
-            if @hide_indicator || no_sort_direction_specified?
+            if @hide_indicator || no_sort_direction_specified? 
```

Comment:
Can you remove the extra whitespace at the end of line 205?

We could shorten this to:

```ruby
def order_indicator
  return if @hide_indicator || no_sort_direction_specified?
  direction_arrow
end
```